File: TouchPOS_API/Models/VoidModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
	public class VoidModels
	{
		public class Void_EWALLET_List
		{
			
			public string BARCODE { get; set; }
			public string MRTSLIPNO { get; set; }
			public int SEQ { get; set; }
			public string foodid { get; set; }
			public string FOODNAME { get; set; }
			public  double MRTAMOUNT { get; set; }
			public string CARDTENDOR { get; set; }
			public string CARDTENDOR_NAME { get; set; }
			
			public Void_EWALLET_List()
            {
				SEQ = 0;
				foodid = "";
				FOODNAME = "";
			}

		}

		public class SALE_void
        {
			public string SHOPID { get; set; }
			public string BARCODE { get; set; }
			public string MRTSLIPNO { get; set; }
			public int SEQ { get; set; }
			public string foodid { get; set; }
			public string FOODNAME { get; set; }
			public string CARDTENDOR { get; set; }
			public string Voidtype { get; set; }
			
			public string CN_DOCNO { get; set; }
			public string VOIDNO { get; set; }
		}





	}
}

File: TouchPOS/Models/VoidModels.vb
Imports MD
Public Class VoidModels
    Public Class value
        Public Respon As New ResponseModels
		Public Data As List(Of Void_EWALLET_List)
	End Class

	Public Class Void_EWALLET_List


		Public Property BARCODE As String
		Public Property MRTSLIPNO As String
		Public Property SEQ As Integer
		Public Property foodid As String
		Public Property FOODNAME As String
		Public Property MRTAMOUNT As Double
		Public Property CARDTENDOR As String
		Public Property CARDTENDOR_NAME As String


	End Class
	Public Class SALE_void

		Public Property SHOPID As String
		Public Property BARCODE As String
		Public Property MRTSLIPNO As String
		Public Property SEQ As Integer
		Public Property foodid As String
		Public Property FOODNAME As String
		Public Property CARDTENDOR As String
		Public Property CARDTENDOR_NAME As String
		Public Property Voidtype As String
		Public Property CN_DOCNO As String
		Public Property VOIDNO As String

	End Class

End Class
